src/Routes.js

```diff
@@ -6,14 +6,52 @@ import React from "react";
 import { Route, Switch } from "react-router-dom";
 
 //> Components
```

Comment:
```suggestion
//> Components
/**
 * HomePage: Landing Page
 * ProfilePage: User Profile Page
 * CompanyPage: Company Profile Page
 * TalkPage: Global Talk Page
 * SettingsPage: Profile Settings Page
 */
```

I think it would be a good idea to rename ProfilePage page to UserPage.

Reply:
We decided to not use this style of commenting.